coalib/results/ResultFilter.py

```diff
@@ -210,15 +211,28 @@ def remove_result_ranges_diffs(result_list, file_dict):
 
 def ensure_files_present(original_file_dict, modified_file_dict):
     """
-    Ensures that all files are available as keys in both dicts.
+    Ensures that all files are available as keys in both dicts. Return a
+    dictionary of renamed files.
 
     :param original_file_dict: Dict of lists of file contents before  changes
     :param modified_file_dict: Dict of lists of file contents after changes
     """
     affected_files = set(original_file_dict.keys()).union(
         set(modified_file_dict.keys()))
+    original_unique_files = affected_files - set(modified_file_dict.keys())
```

Comment:
I think it'd be good to do something like:

```
original_files = set(original_file_dict.keys())
modified_files = ...
affected_files = original_files | modified_files
original_unique_files = original_files - modified_files
```

and later instead of `if file not in original_file_dict` you can just `if file not in original_files` as the set might be better optimized for that operation